test/network/BancorNetwork.ts

```diff
@@ -3910,6 +3926,8 @@ describe('BancorNetwork Financial Verification', () => {
         bntTradingLiquidity: Decimal;
         averageRateN: Decimal;
         averageRateD: Decimal;
+        invAverageRateN: Decimal;
+        invAverageRateD: Decimal;
```

Comment:
`averageInvRate` would probably be more accurate (otherwise, it's just the inverse of the original average rate)

Reply:
👍